enhancements/accelerators/out-of-tree-drivers.md

```diff
@@ -0,0 +1,974 @@
+---
+title: Supporting out-of-tree drivers on OpenShift
+  - "@zvonkok"
+reviewers:
+  - "@ashcrow"
+  - "@darkmuggle"
+  - "@cgwalter"
+approvers:
+  - "@ashcrow"
+  - "@cgwalters"
+  - "@darkmuggle"
+creation-date: 2020-04-03
+last-updated: 2020-07-21
+status: provisional
+see-also:
+  - "/enhancements/TODO.md"
+replaces:
+  - "/enhancements/TODO.md"
+superseded-by:
+  - "/enhancements/TODO.md"
+---
+
+# Supporting out-of-tree drivers on OpenShift
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+1. Should SRO be CVO or OLM managed, SRO creates an ClusterOperator object for must-gather and better status reporting to customers/developers/users
```

Comment:
@zvonkok the CO is most importantly needed to prevent upgrades that would render the special resource unusable due to missing drivers for the new kernel version, right? Can you note that here?

Reply:
This should be removed as we agreed this should be OLM managed.